modules/javafx.controls/src/test/java/test/javafx/css/PropertySizeTest.java

```diff
@@ -216,9 +216,10 @@ public void relativeFontSizeSetOnAllNestedParentsAndControlsTest() {
         p3.setStyle("-fx-font-size: 0.6em");
         p4.setStyle("-fx-font-size: 0.5em");
 
-        // When a Parent and its Parent's font size is relative then that Parent's
-        // font size is computed in relative to it's grand parent's font size.
-        // For compatibility we are preserving this odd behaviour.
+        // Ideally relative font size of a parent should be relative to font size of its parent.
```

Comment:
This is a bit confusing. I think you are saying that the relative size of a (node's) parent should be relative to the size of _that parent's_ parent, right? If so, then maybe something like this?

// Ideally the relative font size of a parent should be relative to the font size of that parent's parent.